### PR TITLE
Increases the spin and flip cooldowns

### PR DESCRIPTION
**Pull request by Sheits**
Its 10 seconds now :)
You can still get dizzy and trip from spamming hotkeys

### CODE REVIEW

**Comment by maxspells** [comment]
How am I supposed to spin flip clap if I can't spin flip clap

File: fulp_modules/Z_edits/emote_edits/cooldowns.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/*
+ *	# Custom emote cooldowns
+ *
+ *	Users will still trip and get dizzy/vomit from spamming spin and flip
+ */
+
+/datum/emote/flip
+	cooldown = 10 SECONDS
```
Comment (JohnFulpWillard) [inline fulp_modules/Z_edits/emote_edits/cooldowns.dm:8]: This isn’t what I told you to do

Comment (Sheits) [reply fulp_modules/Z_edits/emote_edits/cooldowns.dm:8]: you didnt tell me to do anything :/

**Comment by ATH1909** [comment]
bruh

**Pull request merged**